models/user.js

```diff
@@ -6,16 +6,34 @@ const { ObjectId } = mongoose.Schema.Types
 const userSchema = new mongoose.Schema({
   username: {
     type: String,
-    unique: true
+    unique: true,
+    minlength: [3, 'minimipituus 3'],
+    maxlength: 200
+  },
+  password: {
+    type: String,
+    minlength: [6, 'minimipituus 6'],
```

Comment:
Nää mongo validaattorit poislukien toi unique validaattori kantsii poistaa koska kaikilla vanhoilla käyttäjillä nää ei välttämättä täyty ja se voi aiheuttaa jotain ongelmia jos niitten pitäis päästä muokkaamaan esim. sähköpostiosoitettaan.

Reply:
no eipä näillä varsinaisesti ole oikeaa käyttöä, että voihan nämä poistaa.
Eikö luulisi että päivittäessä pakottaisi vain vaadittuun muotoon, eikai tämä jo tallennettuun tietoon kajoa?